packages/daemon/src/types.d.ts

```diff
@@ -385,6 +448,15 @@ export type EndoWebBundle = {
   powers: ERef<unknown>;
 };
 
+export type EndoBootstrap = FarRef<{
```

Comment:
Although there is no `EndoBootstrap`, let's do this for consistency with `FarEndoReadable`?
```suggestion
export type FarEndoBootstrap = FarRef<{
```